src/js/utils/faye.js

```diff
@@ -38,7 +39,9 @@ export function initFaye() {
         });
 
         return faye.subscribe(`/v1/conversations/${state.conversation._id}`, (message) => {
-            store.dispatch(addMessage(message));
+            if (!message.source.id || message.source.id !== getDeviceId()) {
```

Comment:
I don't think you need to check for `message.source.id` since it should be different than the device id anyway